packages/ui/src/1_atoms/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactNode, FC } from 'react';
+
+import classNames from 'classnames';
+
+import { Heading } from '../Heading/Heading';
+import { HeadingType } from '../Heading/Heading.types';
+import { Icon } from '../Icon/Icon';
+import { IconNames } from '../Icon/Icon.types';
+import styles from './Accordion.module.css';
+
+export interface IAccordionProps {
+  label: string | ReactNode;
```

Comment:
```suggestion
  label: ReactNode;
```
`string` is included in `ReactNode`.